ds701_book/21-Networks-I.qmd

```diff
@@ -253,422 +387,409 @@ nx.draw_networkx(DG,
+
+This is the probability that neighbors are connected for a __random node__.
 
-This is the probability that neighbors are connected for a __random node.__
+In the previous example, $C^{(2)} = \frac{1}{5} \left(1 + 1 + \frac{1}{6}\right) = \frac{13}{30} = 0.433.$
 
-$$ C^{(2)} = \frac{1}{5} (1 + 1 + \frac{1}{6}) = \frac{13}{30} = 0.433$$
+---
 
 What is the clustering coefficient in a $G(n, p)$ random graph?
 
-This is easy: the probability that two of your neighbors are connected is the same as the probability that __any__ two nodes are connected.
-    
-In other words, $C^{(1)} = C^{(2)} = p$.
+The probability that two of your neighbors are connected is the same as the probability that __any__ two nodes are connected, i.e., $C^{(1)} = C^{(2)} = p$.
 
-__Real World Graphs Tend to Show Large Amounts of Clustering.__
+## Real World Graphs 
 
-In practice one finds that real world graphs show strong clustering.
+In practice, real world graphs show strong clustering.
 
-For example, consider a social network.  Your friends are much more likely to be themselves friends than two randomly chosen people.
+For example, consider a social network. Your friends are much more likely to be themselves friends than two randomly chosen people.
 
-<center>
-    
-<img src="figs/L21-clustering-coefs.png" alt="Figure" width="800px">
-    
-</center>
+![](figs/L21-clustering-coefs.png){fig-align=center width=800px}
 
 ## Clustering and Path Length: Small Worlds
 
-The strong presence of clustering in networks leads to a question: how long is a typical shortest path between nodes?
+The strong presence of clustering in networks leads to a question. 
+
+How long is a typical shortest path between nodes?
     
-The average shortest path length between nodes is one way to measure a network's "diameter."
+The average shortest path length between nodes is one way to measure a network's *diameter*.
 
-Let's consider a highly clustered graph:
+---
 
-<center>
-    
-<img src="figs/L21-high-clustering.png" alt="Figure" width="300px">
-    
-</center>
+Let's consider a highly clustered graph.
 
-In this graph, each node has four neighbors, and of the 6 pairs of neighbors, 3 are connected.
+![](figs/L21-high-clustering.png){fig-align=center width=300px}
 
-So the clustering coefficient is 0.5, quite high.
+In this graph, each node has 4 neighbor. Of the 6 pairs of neighbors, 3 are connected, yielding a clustering coefficient of 0.5. This is quite high.
 
 What is the average shortest path between nodes?   
 
-It seems to be around $n/8$ - on average you go 1/4 of the way around the circle, in hops of 2.
+It seems to be around $n/8$. On average you go 1/4 of the way around the circle, in hops of 2. 
 
-So the path length grows linearly with $n$. 
+In this example, the path length grows linearly with $n$. 
 
-__If the number of nodes is large, the average path length is large.__
+In general, __if the number of nodes is large, the average path length is large.__
```

Comment:
Just for this example? Or in general for any network?

Reply:
I clarified that it is for this example. It is not true in general. Good catch.